MovieMVVM/ServiceLayer/NetworkService/NetworkService.swift

```diff
@@ -86,29 +73,24 @@ final class NetworkService: NetworkServiceProtocol {
         }.resume()
     }
 
-    func getBaseUrl(currentCategoryMovies: String) -> String {
-        guard let APIKeyValue = keychainService.getKey(forKey: KeychainKey.apiKey) else { return Constants.emptyText }
-        return "\(Constants.themoviedbQueryText)\(currentCategoryMovies)\(Constants.apiKeyQueryText)" +
-            "\(APIKeyValue)"
-        "\(Constants.languageText)\(Constants.pageQueryText)\(Constants.pageQueryText)"
-    }
-
-    // MARK: - Private Methods
-
-    private func getCategoryURL(categoryMovies: CategoryMovies) -> String {
-        var currentCategoryMovies = getCategoryString(categoryMovies: categoryMovies)
-        let urlString = getBaseUrl(currentCategoryMovies: currentCategoryMovies)
-        return urlString
-    }
-    
-    private func getCategoryString(categoryMovies: CategoryMovies) -> String {
+    func getCategoryURL(categoryMovies: CategoryMovies) -> String {
+        var currentCategoryMovies = Constants.emptyText
         switch categoryMovies {
         case .topRated:
-            return Constants.topRatedQueryText
+            currentCategoryMovies = Constants.topRatedQueryText
         case .popular:
-            return Constants.popularQueryText
+            currentCategoryMovies = Constants.popularQueryText
         case .upcoming:
-            return Constants.upcomingQueryText
+            currentCategoryMovies = Constants.upcomingQueryText
         }
+        let urlString = getBaseUrl(currentCategoryMovies: currentCategoryMovies)
+        return urlString
+    }
+
+    func getBaseUrl(currentCategoryMovies: String) -> String {
+        guard let APIKeyValue = keychainService.getKey(forKey: KeychainKey.apiKey) else { return Constants.emptyText }
+        return "\(Constants.themoviedbQueryText)\(currentCategoryMovies)\(Constants.apiKeyQueryText)" +
+            "8216e974d625f2a458a739c20007dcd6"
```

Comment:
```suggestion
            ""
```
 в константы + Интерполяция